src/DisputeManager.ts

```diff
@@ -41,7 +41,8 @@ export function handleNewDispute(event: NewDispute): void {
   updateRound(event.params.disputeId, dispute.lastRoundId, event)
   tryDecodingAgreementMetadata(dispute)
 
-  ArbitrableTemplate.create(event.params.subject)
+  ArbitrableTemplate_v1_1_2.create(event.params.subject)
+  ArbitrableTemplate_v1_1_3.create(event.params.subject)
```

Comment:
I don't really understand this, but did we change the `IArbitrable` interface?

Also, wouldn't these two have the same object id and clash?

Reply:
Yes, we did change it, see [here](https://github.com/aragon/aragon-court/pull/268)

This is clearly an ugly hack to support both Arbitrables. However, we are safe to do it since the only change we introduced was a new parameter in the event, which means a new signature. Then, only one of those templates will track the evidence for the given subject, the other one will be just sitting there. The bad thing is that we cannot distinguish, based on a subject, what interface it supports since the interface ID has not changed. 

For example, I already checked that both alternatives are working correctly:
- Dispute with new evidence format: [#26](https://court-staging.aragon.org/disputes/26)
- Dispute with old evidence format: [# 15](https://court-staging.aragon.org/disputes/15)

Note that this hack will only fail if there is one Arbitrable emitting both event formats. 
I think at some point we will deprecate the old format, but it means probably losing all the current disputes' evidences. 

Another way of doing it could be having an internal config based on block numbers per network, so we use the old format for a given set of blocks.

Reply:
Right, I remember now.

I was thinking about "mashing" the two ABIs together, since it'd just be inserting a new event into the ABI, but the deprecation date for this particular case sounds better.

Perhaps another thing we can do is hardcode a list of contracts on each network that use the deprecated interface?

Reply:
I like the idea of mashing the two ABIs, having a list of contracts in the code is a PITA, we are already planning to do that for the court subscriptions upgrade but in that case, both versions are totally incompatible. 

I'll push a PR to improve this hack soon